config/servers.yml

```diff
@@ -71,6 +71,8 @@
 #      brokers:                                 localhost:9092
 #      zkAddress:                               localhost:2181
 #      mode:                                    embeddedHeaders
+#      headers:
+            # comma-delimited list of additional (string-valued) header names to transport
```

Comment:
They no-longer are limited to string-valued - we switched to JSON [here](https://github.com/spring-projects/spring-xd/pull/1408).